CHANGELOG.md

```diff
@@ -3,6 +3,11 @@ All notable changes to the Azure Pipelines extension will be documented in this
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/). Versioning follows an internal Azure DevOps format that is not compatible with SemVer.
 
+## 1.157.2
+### Added
+- Added troubleshooting steps in README
+- Added more telemetry
```

Comment:
Not sure this is worth changelogging.

Reply:
(Troubleshooting steps is, though.)

Reply:
Will keep the first point in changelog. Removing the telemetry point.